macros/edr/tests/test_collect_metrics.sql

```diff
@@ -0,0 +1,30 @@
+{% test collect_metrics(
```

Comment:
Should we support specifying custom metrics?

Reply:
No, as in `null_count`, `max_length`, etc.
Like the `column_anomalies` parameter in `column_anomalies` to include/exclude metrics [here](https://github.com/elementary-data/dbt-data-reliability/blob/dd437b854464bdc489539409d5b9b5d00f91821f/macros/edr/data_monitoring/monitors_query/column_monitoring_query.sql#L59).